app/src/components/Shared/ToolTip.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import ToolTipStyle from '../../../styles/components/c-tooltip-info.scss';
+import ImgWhite from '../../../assets/icons/info_white.svg';
+import ImgBlack from '../../../assets/icons/info_black.svg';
+
+class ToolTip extends Component {
+
+  render() {
+    return (
+      <Link
+        to="/definitions"
```

Comment:
https://www.pivotaltracker.com/story/show/128939267

When the user clicks this, they need to be taken to the "definitions" page, with "fishing effort" open and autoscroll to it.